app/models/project/queue.rb

```diff
@@ -0,0 +1,69 @@
+class Project::Queue
+  def initialize(project, current_user, session)
+    @project = project
+    @current_user = current_user
+    @session = session
+  end
+
+  def next
+    find_current || sample
+  end
+
+  def current(info_request_id)
```

Comment:
I ummed and ahhed about this a little.

I started my test like:

```ruby
  describe '#current' do
    subject { queue.current = 1 }
    it { is_expected.to eq(1) }
  end
```

…and it was immediately passing when I implemented the method stub:

```ruby
def current=(info_request_id_
end
```

TIL that `=` methods always return the given value:

```
class Foo
  def bar=(x)
  end

  def bar2=(x)
    x * 20
  end

  def baz(x)
  end
end

f = Foo.new
f.bar = 1
# => 1

f.bar2 = 1
# => 1

f.baz(1)
# => nil
```

So basically my test is doing nothing.

Could dive in to the internals of `session` but that shouldn't really be exposed outside the class.

Anyway, the current test isn't doing much as mentioned, so could implement this as `current=` since the behaviour is tested through `#next`